samples/amd-token-generation/spin.toml

```diff
@@ -0,0 +1,23 @@
+spin_manifest_version = 2
+
+[application]
+authors = ["Radu Matei <radu@fermyon.com>"]
+description = ""
+name = "amd-token-generation"
+version = "0.1.0"
+
+[variables]
+encryption_key = { required = true }
```

Comment:
should this be a secret?

Reply:
Note that `secret = true` doesn't actually do anything today. It was meant to allow partial debug logging of variable values but in instead we just never log any. :shrug:

Reply:
I suppose optimistically some external tooling could take it into account :sweat_smile: